test/filters.js

```diff
@@ -60,7 +60,7 @@ test('Adding and removing a Image Mask filter', async t => {
 
 test('Adding and removing a Crop Pad filter', async t => {
   const app = t.context.app;
-  const sourceName = 'Color Source';
+  const sourceName = 'Color Source 3';
```

Comment:
Why this change is needed?

Reply:
just easy logic to force the creation of a new source